index.html

```diff
@@ -54,7 +58,123 @@
+<nav>
+  <h3>main site menu</h3>
+  <ul>
+    <lI><a href="#">Home Page</a></lI>
+    <li><a href="#">About us</a></li>
+    <li><a href="#">Contact us</a></li>
+  </ul>
+</nav>
+</header>
+ <main>
 
-    <h1>Assignment 01 - To Be Completed</h1>
+  <section>
+  <h1>Ottawa's number 1 Electronics retailer</h1>
+  <img src="ottawa lightening.jpeg" alt="lightening strikes downtown Ottawa">
+  </section>
+
+  <Section>
+<h2>costumer testimonials</h2>
+  <p>"This store has the best  costumer service of all time.
+  the moment I walked in the store, was the single most joyus moment of my existence.11/10"
+  
+  -Jane Doe
+  </p>
+
+  <P>
+    "very reasonable pricing, my son loves his brand new gamestation 5. 8/10"  
+    -Bob Stan
+  </P>
+
+  <p>
+    "wonderful stores, I truly love the wild variety of products. 9/10"
+    -Tom Ryan
+  </p>
+</section>
+
+<section>
+  <h2> Products</h2>
+  <H3>Gamestation 5</H3>
+<img src="GS5.webp" alt="Gamestation 5">
+
+<P>
+  Gamestation 5 is an all new hightech gaming console. for the price of only $5,000, you can reach the pinnacle of gaming in your living room.
+</P>
+  <H3>PC2</H3>
+  <img src="pc2.jpeg" alt="pc2">
+  the long awaited sequal to the classic "personal computer", 
+  is now availabl exclusively at our store. purchase now for $9999.99
+  don't worry if that price seems a little low, we have plenty of upgraded components for you to buy and install!) 
+  <h3>6G router</h3>
+  <img src="6g .jpeg" alt="6g router">
+  <p>
+    coming soon. 6g network speeds are approaching faster then you can imagine. get ahead of the game
+    and pre order from a selection of 6g compatable routers. starting at the bargin bin price of $599.99!
+  </p>
+</section>
+
+<section>
+  <h2> Repair Services</h2>
+
+  <P>
+    we offer a variety of electronic repairs, with a range of prices depending on the details of your needs.
+  </P>
+<ul>
+  <li>computers/laptops</li>
+  <p>$200-1000</p>
+  <li>phones</li>
+  <p>$80-350</p>
+  <li>speakers/headphones</li>
+  <p>$50-300</p>
+  <p> TV/displays</p>
+  <P>$200-2000</P>
+</ul>
+
+<P>*please note replacement parts not included in provided price ranges*</P>
+</section>
+
+<section>
+  <h3>Sign up for our Repair service through email!</h3>
+  <p>
+    fill out the form below and we'll get back to you within the next 3 weeks.
+<form>
+  <div>
+  <label>User's Name</label>
+  <input type="text">
+  </div>
+  <div>
+    <label>User's Email address</label>
+    <input type="text">
+  </div>
+  <div>
+    <label>Device in need of repair</label>
+    <input type="text">
+  </div>
+  <div>
+  <label>Additional information</label>
+  <input type="text">
+  </div>
+  <button>submit</button>
+ 
+</form>
+</section>
+ </main>
+
+ <footer>
+  
+  <img src="lightning-logo.png" alt="ottawa electronics logo">
+<nav>
+  <h2>Ottawa Electronics</h2>
+<ul>
+  <lI><a href="#">Home Page</a></lI>
+ <li><a href="#">About us</a></li>
+  <li><a href="#">Contact us</a></li>
+</ul>
+</nav>
+ </footer>
```

Comment:
Missing the copywrite notice 

Also, have a look at the tutorial exercise code samples, as I think this would be better with the h2 inside the footer but not inside the nav